scss/nacara.scss

```diff
@@ -63,3 +63,26 @@ body {
 .icon+span {
     margin-left: .5rem;
 }
+
+#toc-column {
+    // TODO: I'd like to reduce the top margin in mobile
+    // But the jump when changing to content is noticeable
```

Comment:
I didn't try the code.

But if you want it to be less abrupt you can animate the property.